ruby.c

```diff
@@ -319,80 +319,81 @@ usage(const char *name, int help, int highlight, int columns)
-        M("--parser={parse.y|prism}",           ", --parser=prism",
-          "the parser used to parse Ruby code (experimental)"),
-        M("--backtrace-limit=num",                  "", "limit the maximum length of backtrace"),
-        M("--verbose",                              "", "turn on verbose mode and disable script from stdin"),
-        M("--version",                              "", "print the version number, then exit"),
-        M("--crash-report=TEMPLATE",                "", "template of crash report files"),
-        M("-y",                          ", --yydebug", "print log of parser. Backward compatibility is not guaranteed"),
-        M("--help",			            "", "show this message, -h for short message"),
+        M("--backtrace-limit=num",        "",            "Set backtrace limit."),
+        M("--copyright",                  "",            "Print Ruby copyright."),
+        M("--crash-report=template",      "",            "Set template for crash report file."),
+        M("--disable=features",           "",            "Disable features; see list below."),
+        M("--dump=items",                 "",            "Dump items; see list below."),
+        M("--enable=features",            "",            "Enable features; see list below."),
+        M("--external-encoding=encoding", "",            "Set default external encoding."),
+        M("--help",		          "",            "Print long help message; use -h for short message."),
+        M("--internal-encoding=encoding", "",            "Set default internal encoding."),
+        M("--parser=parser",              "",            "Set Ruby parser: parse.y or prism."),
+        M("--verbose",                    "",            "Set $VERBOSE to true; ignore input from $stdin."),
+        M("--version",                    "",            "Print Ruby version."),
+        M("-y",                           ", --yydebug", "Print parser log; backward compatibility not guaranteed."),
     };
     static const struct ruby_opt_message dumps[] = {
-        M("insns",                  "", "instruction sequences"),
-        M("insns_without_opt",      "", "instruction sequences compiled with no optimization"),
-        M("yydebug(+error-tolerant)", "", "yydebug of yacc parser generator"),
-        M("parsetree(+error-tolerant)","", "AST"),
-        M("parsetree_with_comment(+error-tolerant)", "", "AST with comments"),
+        M("insns",                                   "", "Instruction sequences."),
+        M("insns_without_opt",                       "", "Instruction sequences compiled with no optimization."),
+        M("yydebug(+error-tolerant)",                "", "yydebug of yacc parser generator."),
+        M("parsetree(+error-tolerant)",              "", "Abstract syntax tree (AST)."),
+        M("parsetree_with_comment(+error-tolerant)", "", "AST with comments."),
+        M("prism_parsetree",                         "", "Prism AST with comments."),
```

Comment:
I don't think there are comments included in the dump.

```suggestion
        M("prism_parsetree",                         "", "Prism AST."),
```